hamp_radar/iquick.py

```diff
@@ -366,6 +366,47 @@ def decode_iq(rawdata):
 }
 
 
+def single_dspparams_data(data):
+    """
+    Returns raw arrays of data from the first occurrence of DSP parameters
+    up to the next occurrence or up to the end of the data.
+
+    The first value in the returned list of raw arrays is the DSP parameters
+    configuration (tag == "PPAR").
+
+    Raises warning if no PPiAR tags are found in the data.
+
+    Parameters:
+    data (any): The input data (memory map or open file) from which to extract the
+                arrays associated with a single DSP parameter configuration.
+
+    Returns:
+    list: A list of raw arrays extracted from the data.
+    """
+
+    mmbgs = get_geometry(data)
+
+    start = None
+    end = -1
+    for i, mmbg in enumerate(mmbgs):
+        if mmbg.tag == "PPAR":
+            if start is None:
+                start = i
+            else:
+                end = i
+                break
+
+    if start is None:
+        start = 0
+        print(
+            "Warning: No PPar tags found, using data from entire file which isn't associated with any DSP configuration"
```

Comment:
Not sure if there's a correct spelling. But this would make it consistent throughout the file.
```suggestion
            "Warning: No PPAR tags found, using data from entire file which isn't associated with any DSP configuration"
```